conf.py

```diff
@@ -113,12 +75,7 @@
 numfig_format = {
     'table': 'Table %s'
 }
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
Bad config in Sphinx 6+

Reply:
Nice. CI is using Sphinx 7.0 instead of 5.3